diff -urN gconf-editor-2.8.1.orig/data/gconf-editor.desktop.in gconf-editor-2.8.1/data/gconf-editor.desktop.in
--- gconf-editor-2.8.1.orig/data/gconf-editor.desktop.in	2004-10-01 15:02:24.712546672 +0200
+++ gconf-editor-2.8.1/data/gconf-editor.desktop.in	2004-10-01 15:02:42.229883632 +0200
@@ -7,7 +7,7 @@
 Type=Application
 Icon=gconf-editor.png
 StartupNotify=true
-Categories=GNOME;Application;System;
+Categories=GTK;GNOME;System;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gconf-editor
 X-GNOME-Bugzilla-Component=general
